packages/amplication-server/src/core/assistant/assistantFunctions.service.ts

```diff
@@ -284,12 +292,33 @@ export class AssistantFunctionsService {
     const installations = [];
     let authEntityExist = false;
 
+    const service = await this.resourceService.resource({
+      where: {
+        id: serviceId,
+      },
+    });
+
+    const codeGenerator =
+      CODE_GENERATOR_NAME_TO_ENUM[service.codeGeneratorName] ||
+      EnumCodeGenerator.NodeJs;
+
+    if (!service) {
```

Comment:
I would move the `if (!service)` check to be above the `const codeGenerator =` part